cmd/goal/asset.go

```diff
@@ -545,6 +538,13 @@ var sendAssetCmd = &cobra.Command{
 		if err != nil {
 			reportErrorf("Cannot construct transaction: %s", err)
 		}
+		explicitFee := cmd.Flags().Changed("fee")
```

Comment:
Is this change okay for single transactions? I see that in `client.FillUnsignedTxTemplate(sender, firstValid, lastValid, fee, tx)`, there is an explicit check for [`if fee == 0`](https://github.com/algorand/go-algorand/blob/941c8204591b166e68bb205d04ab0eb0b7b318cb/libgoal/transactions.go#L402) to set the fee to the minimum Tx Fee. It looks like this change might override that setting for both single and group transactions.